components/general/BottomNav.jsx

```
@@ -11,21 +11,21 @@ import { Button } from "../ui/button";
 export default function BottomNav({ tabsState, onTabClick }) {
   return (
-    <nav className="absolute gap-3 items-center justify-center bottom-6 flex bg-white/20 rounded-xl backdrop-blur-[30px] border border-white/40">
+    <nav className="absolute gap-3 items-center justify-center bottom-6 flex bg-white/10 rounded-xl backdrop-blur-[30px] border border-white/40">
       <TooltipProvider delayDuration="50">
         <Tooltip>
           <TooltipTrigger>
             <div
               onClick={() => onTabClick("aboutMe")}
-              className="p-3 text-md flex flex-col items-center justify-center gap-1 text-black cursor-pointer"
+              className="p-3 text-md flex flex-col items-center justify-center gap-1 text-white cursor-pointer"
             >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)